modules/insights-operator-using-filters-on-advisor-recommendations.adoc

```diff
@@ -0,0 +1,32 @@
+// Module included in the following assemblies:
+//
+// * support/remote_health_monitoring/using-insights-to-identify-issues-with-your-cluster.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="insights-operator-using-filters-on-advisor-recommendations_{context}"]
+= Using filters to remove unnecessary results from Insights advisor recommendations
```

Comment:
Rename this procedure module to `Applying a filter` to make it simple and clear.